receiver/ciscoosreceiver/internal/metadata/generated_metrics_test.go

```diff
@@ -0,0 +1,31 @@
+// Code generated by mdatagen. DO NOT EDIT.
+
+package metadata
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.opentelemetry.io/collector/receiver/receivertest"
+)
+
+func TestMetricsBuilder(t *testing.T) {
+	tests := []struct {
+		name string
+	}{
+		{
+			name: "default",
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			settings := receivertest.NewNopSettings(receivertest.NopType)
+			mb := NewMetricsBuilder(loadMetricsBuilderConfig(t, tt.name), settings)
+
+			metrics := mb.Emit()
+
+			// Since we removed all metrics, we expect an empty metrics object
```

Comment:
These files (generated_metrics.go and generated_metrics_test.go) are generated. Let's not edit them manually but remove for now